src/protocols/secure_channel/PairingSession.cpp

```diff
@@ -142,6 +142,16 @@ CHIP_ERROR PairingSession::EncodeSessionParameters(TLV::Tag tag, const ReliableM
 
     uint16_t maxPathsPerInvoke = CHIP_CONFIG_MAX_PATHS_PER_INVOKE;
     ReturnErrorOnFailure(tlvWriter.Put(TLV::ContextTag(SessionParameters::Tag::kMaxPathsPerInvoke), maxPathsPerInvoke));
+
+#if INET_CONFIG_ENABLE_TCP_ENDPOINT
+    // Support for both TCP CLient and Server(0x06)
+    uint16_t supportedTransports = 0x06;
```

Comment:
please set bit flags constants and use `|`. This should read like `kTcpClient | kTcpServer` or similar. Do not hardcode magic numbers.